README.md

```diff
@@ -372,20 +370,26 @@ export PATH=$(pwd)/target/testnet:$PATH
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript
+❯ npm run zombie
 
 Usage: zombienet [options] [command]
 
 Options:
-  -p, --provider <provider>                Override provider to use (choices: "podman",
-                                           "kubernetes", default: kubernetes)
+  -c, --spawn-concurrency <concurrency>    Number of concurrent spawning process to launch, default is 1
```

Comment:
make it aligned with current output, [see](https://github.com/paritytech/zombienet/blob/8f007c789ec12d204406db0200778da83bab2b4e/javascript/packages/cli/src/cli.ts#L76)

Reply:
here and others